README.md

```diff
@@ -75,7 +98,9 @@ everything went fine.
 
     $> cd centreon-engine/build
     $> cmake .
-    ...
+    $> cmake" -DWITH_PREFIX=/usr -DWITH_PREFIX_BIN=/usr/sbin -DWITH_USER=centreon-engine -DWITH_GROUP=centreon-engine -DCMAKE_BUILD_TYPE=Debug -DWITH_RW_DIR=/var/lib/centreon-engine/rw -DWITH_PREFIX_CONF=/etc/centreon-engine -DWITH_VAR_DIR=/var/log/centreon-engine -DWITH_PREFIX_LIB=/usr/lib64/centreon-engine -DWITH_TESTING=On -DWITH_SIMU=On -DWITH_CREATE_FILES=OFF -DWITH_BENCH=On ..
```

Comment:
-DCMAKE_BUILD_TYPE=Release

Reply:
-DWITH_SIMU=On is not needed for an end user. Also, -DWITH_BENCH= On is not needed